object.go

```diff
@@ -196,104 +198,6 @@ func (e *Encoder) encObject(v POJO) error {
 
 	return nil
 }
-
-func (e *Encoder) encNotPOJOObject(v interface{}, javaClassName string) error {
```

Comment:
why delete this func?

Reply:
As wongoo commented above, I merge the 'no pojo object' register logic in  encPOJOObject function.